Community/Service Requests/service_request_host/service_request_host_page.py

```diff
@@ -0,0 +1,157 @@
+# Copyright 2018 BlueCat Networks (USA) Inc. and its affiliates
```

Comment:
Please change date to 2019. There are a few instances of 2018 and one of 2017 in the files.

Reply:
@delmeherbert Can you take a look at the latest files? That should have been changed with my last set of commits.